js/map_utils.js

```diff
@@ -185,10 +186,22 @@ function documentCategorySwitch(category, prevCategory, action) {
 }
 
 
+function highlightCircle(currentCircle) {
+  var orig = currentCircle,
+      origNode = orig.node();
+  var duplicate = d3.select(origNode.parentNode.appendChild(origNode.cloneNode(true), 
+                                                            origNode.nextSibling));
+                                                            
+  // style duplicated circles sitting on top
+  duplicate
+    .classed('circle-duplicate', true)
+    .style("pointer-events", "none")
+    .style("opacity", 1); // makes the duplicate circle on the top
+}
```

Comment:
i don't think lines 190 through 200 should be in here following the merge of my PR

Reply:
oh! also, the functions highlightCircle and unhighlightCircle should now live in circles.js, not here in map_utils.js